geode-docs/tools_modules/gfsh/command-pages/start.html.md.erb

```diff
@@ -605,6 +621,14 @@ start server --name=value [--assign-buckets(=value)] [--bind-address=value]
 <td> </td>
 </tr>
 <tr>
+<td><span class="keyword parmname">\-\-membership-bind-address</span></td>
+<td>Specifies the IP address to which the UDP membership-related traffic will be bound. If defined, it has precedence over the value of <code>bind-address</code>. Wildcard addresses are not accepted.
+<div class="note note">
+<b>Note:</b> By default, UDP membership-related traffic is bound to <code>bind-address</code> but there is one exception. In case the server is bound to all local addresses, UDP membership-related traffic will be bound to the local host machine's address by default.
```

Comment:
Delete the note here, which is redundant, as it repeats the description of default behavior already explained in the --bind-address entry above.